model/deployment.go

```diff
@@ -17,39 +17,33 @@ type deployment struct {
 	Args      []string `yaml:"args"`
 }
 
-//Deployment returns a k8 deployment for a cloud native environment
+//Deployment returns a k8 deployment
 func (dev *Dev) Deployment() (*appsv1.Deployment, error) {
-	file, err := os.Open(dev.Swap.Deployment.File)
+	return dev.loadDeployment()
+}
+
+//DevDeployment returns a k8 deployment modified with  a cloud native environment
+func (dev *Dev) DevDeployment() (*appsv1.Deployment, error) {
+	d, err := dev.loadDeployment()
 	if err != nil {
 		return nil, err
 	}
 
-	dec := k8Yaml.NewYAMLOrJSONDecoder(file, 1000)
-	var d appsv1.Deployment
-	dec.Decode(&d)
-
-	d.GetObjectMeta().SetName(dev.Name)
```

Comment:
@pchico83 do you think we should remove the name altogether, and just use the name of the deployment instead?

Reply:
Yeap!